envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/listeners/EnvoyListenersFactory.kt

```diff
@@ -313,10 +317,36 @@ class EnvoyListenersFactory(
                 .build()
     }
 
-    private fun accessLog(accessLogPath: String, accessLogType: String): AccessLog {
-        return AccessLog.newBuilder()
-                .setName("envoy.file_access_log")
-                .setTypedConfig(
+    private fun accessLog(
+        accessLogPath: String,
+        accessLogType: String,
+        accessLogFilterHttpCode: pl.allegro.tech.servicemesh.envoycontrol.groups.AccessLogFilter?
```

Comment:
What do you think about aliasing this or Enovy's one?
```kotiln
import io.envoyproxy.envoy.config.filter.accesslog.v2.AccessLogFilter as EnvoyAccessLogFilter
```

Reply:
Or maybe changing its name to `AccessLogFilterSettings` as it isn't a filter but just settings. Then I'd change the parameter name, `accessLogFilterHttpCode`, to something more adequate.

Reply:
will change it to AccessLogFilterSettings